frontend/src/metabase/lib/colors/colors.ts

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable no-color-literals */
+// NOTE: DO NOT ADD COLORS WITHOUT EXTREMELY GOOD REASON AND DESIGN REVIEW
+// NOTE: KEEP SYNCHRONIZED WITH:
+// frontend/src/metabase/css/core/colors.module.css
+// frontend/src/metabase/styled-components/containers/GlobalStyles/GlobalStyles.tsx
+// frontend/src/metabase/styled-components/theme/css-variables.ts
+export const colors = {
```

Comment:
Used in the SDK package, so extracted here to not pull any other constants/things